docs/standards/net_standards.md

```diff
@@ -0,0 +1,61 @@
+# .NET Standards
+
+At Defra, we use .NET as a secondary technology stack, with **Node.js** being our primary stack. The use of .NET must be justified with an **evidence-based decision**. This document outlines the standards for using .NET within Defra, ensuring alignment with modern practices and Microsoft's best practices.
+
+## Key Principles
+
+1. **Primary Stack Preference**:
+   - **Node.js** is the preferred stack for all new projects.
+   - The decision to use .NET must be supported by clear evidence, such as:
+     - Integration with commodity products that require .NET.
+     - Specific backend service requirements that are better suited to .NET.
+
+2. **Backend Services Only**:
+   - .NET is only used for **backend services** and **integration with commodity products**.
+   - **Frontend applications** must not be developed using .NET.
+
+3. **C# Language**:
+   - All .NET development must use the **C# language**.
+   - Refer to the [C# Standards](csharp_coding_standards.md) for language-specific guidelines.
+
+4. **Long-Term Support (LTS)**:
+   - Always strive to use the latest **LTS version** of .NET.
+   - **.NET Framework** should only be used for maintaining legacy applications. New projects must use **.NET 8+**.
+
+## When to Use .NET Over Node.js
+
+While Node.js is the preferred stack, there are scenarios where .NET may be better suited:
+
+1. **Integration with Microsoft Ecosystem**:
+   - Some parts of the Microsoft ecosystem are easier to integrate with when using .NET. For example:
+     - Dynamics 365, which has strong support for .NET libraries and SDKs.
+
+2. **High-Performance Backend Services**:
+   - .NET is well-suited for CPU-intensive operations, such as:
```

Comment:
I'm nit-picking, but I'd possibly argue that .NET isn't especially well-suited to these operations, it's just that JavaScript is a really _bad_ choice for this kind of thing, so it's a much better option

Reply:
I agree, Something around introduces complexity and making it "hard to diagnose" bugs due to shared state and concurrency issues.

Reply:
Added a line to the Node.js justification.  It does feel a bit out of place, what do you think?